### PR TITLE
Center pagination controls

### LINKED ISSUE #419: "Edit Favorite Stops": Pagination should be centered
... instead of `padding-left`ed. Tappable area should probably also be a bit bigger.

### PR DESCRIPTION
**Pull request by mboneil10**
Closes #419

### CODE REVIEW

**Comment by mboneil10** [comment]
Still need to:
- remove padding from pagination area
- make the tappable area larger

File: src/modals/stop-modal/stop.modal.html [hunk 1]
```diff
@@ -35,5 +35,5 @@
       </ion-checkbox>
     </ion-item>
   </ion-list>
-  <pagination-controls center class="pages" (pageChange)="p = $event"></pagination-controls>
```
Comment (mboneil10) [inline src/modals/stop-modal/stop.modal.html:38]: Idk what this `center` is for... didn't seem to center the pagination

**Comment by mboneil10** [comment]
@akaplo, should I make a new `scss` file to remove padding and make a specific class for styling the pagination thing? I don't know what the best approach would be in terms of ionic. edit: yes, that's what is expected of the user. I can read again.

**Comment by mboneil10** [comment]
I'll just do it anyway. That's what PRs are for!

File: src/modals/stop-modal/stop.modal.html (hunk 1 above)
Comment (mboneil10) [reply src/modals/stop-modal/stop.modal.html:38]: Oops, someone meant `text-center` (or this syntax has been deprecated, etc.). Now it "works".

**Comment by akaplo** [comment]
Go for it! You def don't need a _new_ `.scss` file - we have one for each page plus a global one already, so you can just make your edits in the appropriate place.

**Comment by mboneil10** [comment]
Thanks @akaplo

**Comment by mboneil10** [comment]
Idk what's happening with this. I can't apply style in [the way that the guide tells us to](https://github.com/michaelbromley/ngx-pagination#styling), and I've already tried every variation of using the different classes that pagination has. What I've done is put the "proper" syntax up here for anyone to look at.  (Although I don't think that does much).

Should we still focus on getting the tappable area larger?

**Comment by akaplo** [comment]
This is a tough one indeed. I've played around with your changes, but also with no real luck.  What you're doing seems correct to me.

**Comment by mboneil10** [comment]
I'll close this since I couldn't figure it out. Maybe someone else will be able to.

**Pull request closed without merging**